tasks/install-manageiq-conversion-host.yml

```diff
@@ -6,7 +6,7 @@
     user: "{{ manageiq_username }}"
     password: "{{ manageiq_password }}"
     method: "POST"
-    validate_certs: true
+    validate_certs: false
```

Comment:
This does not belong here. Please rebase once #18 is merged.

Reply:
Done.

Reply:
Also added same behavior for RHV conversion host. Had missed it.